Model/ContentManager.php

```diff
@@ -205,4 +207,63 @@ public function remove($content)
 
         $this->em->flush();
     }
+
+    /**
+     * @param Content $content
+     * @param NestedValue $nested_in
+     */
+    public function duplicate(Content $content, NestedValue $nested_in = null)
```

Comment:
This `Content` type-hint should be `ContentInterface` (`Opifer\ContentBundle\Model\ContentInterface`)